pkgs/applications/networking/cluster/tilt/default.nix

```diff
@@ -1,24 +1,23 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, buildGoModule, fetchFromGitHub }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "tilt";
   /* Do not use "dev" as a version. If you do, Tilt will consider itself
-     running in development environment and try to serve assets from the
-     source tree, which is not there once build completes.  */
-  version = "0.11.3";
+    running in development environment and try to serve assets from the
+    source tree, which is not there once build completes.  */
+  version = "0.17.0";
 
   src = fetchFromGitHub {
-    owner  = "windmilleng";
+    owner  = "tilt-dev";
     repo   = pname;
     rev    = "v${version}";
-    sha256 = "035czgr0rn6gcv24vnlr35n9yvy0fwq4spdzsc76gfxckcbcmzz0";
+    sha256 = "0bd01fmrf17njzf8ri4bw4qi7bxcvd3dx7yyf42qfvnp7hrfzipk";
   };
-
-  goPackagePath = "github.com/windmilleng/tilt";
+  vendorSha256 = "18iiy5fv2jk07k8lfj6h60ldkm8i2zkhz61pf1ynw30jvh1yk93r";
```

Comment:
Perhaps add `deleteVendor=true` as per [comment](https://github.com/NixOS/nixpkgs/pull/94827#issuecomment-672144311) :thinking:

This will make `buildGoModule` fetch dependencies inside `go.mod`

Reply:
Was hoping this would work fine on other machines! It seems `deleteVendor` is only needed when building the first time after moving from `buildGoPackage`. It's not the most ergonomic since it pulls the dependencies every build, hopefully the cache speeds this up though.

`nixpkgs-review pr 94827` seems to work for me after adding `deleteVendor = true`, how's it looking on your machine?

Reply:
It builds and runs fine now :ok_hand:

Reply:
This seems to build without `deleteVendor` using just `vendorSha256 = null`?

Reply:
@zowoq, either will work, the difference is where dependencies will be sourced. When `vendorSha256 = null` the vendor folder supplied with the source is used, as opposed to `deletedVendor = true`  where dependencies will be fetched via `go.mod`.

Reply:
`deleteVendor` should only be used if the deps are incomplete/broken.

Reply:
Ah, I wasn't aware, there's no indication in the docs that it should be used as such [source](https://github.com/nixos/nixpkgs/blob/cbab6729cdaa42d23d6a54832de6c0e4bb67d6ed/pkgs/development/go-modules/generic/default.nix#L21-L22).

However `tilt`'s [Makefile](https://github.com/tilt-dev/tilt/blob/3a50ff29915de426739f9e9fbe859291fc9c3ba2/Makefile) makes use of the vendored dependencies, so we should align (I should have checked that to start with).

Reply:
Makes sense, thanks for clearing that up @zowoq, just pushed the change.